zabbix_auto_config/processing.py

```diff
@@ -445,6 +446,13 @@ def enable_host(self, db_host):
         else:
             logging.info("DRYRUN: Enabling host: '%s'", hostname)
 
+    def tag_host(self, zabbix_host):
+        if not self.dryrun:
+            self.api.host.update(hostid=zabbix_host["hostid"], tags=zabbix_host["tags"])
```

Comment:
This will enforce the given tags and only the given tags? Is this wanted? Do a merge instead?